parsec/core/fs/storage/workspace_storage.py

```diff
@@ -421,13 +439,120 @@ async def mark_prevent_sync_pattern_fully_applied(self, pattern: Pattern[str]) -
         await self.manifest_storage.mark_prevent_sync_pattern_fully_applied(pattern)
         await self._load_prevent_sync_pattern()
 
+    async def chunk_storage_get_local_chunk_ids(self, chunk_id: List[ChunkID]) -> List[ChunkID]:
+        return await self.chunk_storage.get_local_chunk_ids(chunk_id)
+
+    async def block_storage_get_local_chunk_ids(self, chunk_id: List[ChunkID]) -> List[ChunkID]:
+        return await self.block_storage.get_local_chunk_ids(chunk_id)
+
     # Vacuum
 
     async def run_vacuum(self) -> None:
         # Only the data storage needs to get vacuuumed
         await self.data_localdb.run_vacuum()
 
 
+_PyWorkspaceStorage = WorkspaceStorage
+if not TYPE_CHECKING:
+    try:
+        from libparsec.types import WorkspaceStorage as _RsWorkspaceStorage
+    except:
+        pass
+    else:
+
+        class WorkspaceStorage:
```

Comment:
better to define this before the `if not typechecking`, this save a useless level of indentation, eg:
```python
class SecretKey:
    [...]

_PySecretKey = SecretKey
if not TYPE_CHECKING:
    try:
        from libparsec.hazmat import SecretKey as _RsSecretKey
    except ImportError:
        pass
    else:
        SecretKey = _RsSecretKey
```

Reply:
Changed